app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert split_integer(17, 4) == [4, 4, 4, 5]
```

Comment:
i think in this test you should check if sum of parts is equal to value. now you just compare the result of the function with the list of parts